### PR TITLE
Expand readme

### PR DESCRIPTION
**Pull request by CriesofCarrots**
@mvines I started a "Customizing the Program" section at the bottom, but I think it could use some information about https://github.com/solana-labs/solana/tree/master/programs/bpf/c/sdk that you might be better equipped to write.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,35 +6,42 @@
 # Tic-Tac-Toe on Solana
 
 This project demonstrates how to use the [Solana Javascript API](https://github.com/solana-labs/solana-web3.js)
-to implement an interactive tic-tac-toe game between two users.
+to build, deploy, and interact with programs on the Solana blockchain, implementing an interactive tic-tac-toe game between two users.
+To see the final product, go to https://solana-example-tictactoe.herokuapp.com/ and wait for another player to join.
+(Simply direct a 2nd browser window to the web app to play against yourself.)
```
Comment (mvines) [inline README.md]:
```suggestion
(Simply direct a second browser window to the web app to play against yourself.)
```

File: README.md [hunk 2]
```diff
@@ -6,35 +6,42 @@
...
 
-* The on-chain portion is a BPF program written in C, see `program-bpf/`
-* Command-line and web front-ends are provided under `src/`
+The project comprises:
```
Comment (mvines) [inline README.md]:
```suggestion
The project comprises of:
```

Comment (CriesofCarrots) [reply README.md]: Grammar says either "comprises" or "is comprised of"

Comment (mvines) [reply README.md]: Oh probably, I almost failed english :)

**Comment by mvines** [comment]
I'll add a bit to "Customizing the Program" once this lands

**Pull request merged**